Trait IntoRustByName<Uuid> not implemented for Row

I'm trying to select an UUID field as shown in this [example](https://github.com/AlexPikalov/cdrs/blob/master/examples/all.rs#L876), using the same crate version for **uuid**:

```rust
let _: Uuid = row.r_by_name("id").expect("my_uuid");
```

But I get this error:

```
error[E0277]: the trait bound `cdrs::types::rows::Row: cdrs::types::IntoRustByName<uuid::Uuid>` is not satisfied
   --> src\db.rs:106:25
    |
106 |       let _: Uuid = row.r_by_name("id").expect("my_uuid");
    |                         ^^^^^^^^^ the trait `cdrs::types::IntoRustByName<uuid::Uuid>` is not implemented for `cdrs::types::rows::Row`
    |
    = help: the following implementations were found:
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<f64>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<std::net::IpAddr>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<uuid::Uuid>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<cdrs::types::map::Map>>
            and 12 others
```

My toolchain:

```
nightly-x86_64-pc-windows-msvc (default)
rustc 1.27.0-nightly (ac3c2288f 2018-04-18)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "cdrs"
-version = "1.2.2"
+version = "1.2.3"
 authors = ["Alex Pikalov <alex.pikalov.khar@gmail.com>"]
 
 description = "Cassandra DB driver written in Rust"
@@ -29,7 +29,7 @@ r2d2 = "0.8.2"
 rand = "0.4.1"
 snap = "0.2.3"
 time = "0.1.38"
-uuid = "0.6.1"
+uuid = "0.6.3"
 
 [dev-dependencies]
 env_logger = "0.4.3"

